doc/src/guides/solving/solve-polynomial.md

```diff
@@ -0,0 +1,517 @@
+# Solve (Find the Roots of) a Polynomial Algebraically or Numerically 
+
+Use SymPy to solve (find the roots of) a univariate polynomial algebraically.
+For example, solving $ax^2 + bx + c = 0$ for $x$ yields $x =
+\frac{-b\pm\sqrt{b^2 - 4ac}}{2a}$.
+
+## Alternatives to Consider
+
+- If you need a numeric (rather than algebraic) solution, you can use either
+    - NumPy's {external:func}`~numpy.roots`
+    - SciPy's {external:func}`~scipy.optimize.root`
+- If you need to solve a multivariate polynomial algebraically,
+    - for simple polynomials, use {func}`~.solve`
+    - for more complex polynomials, try {func}`~sympy.polys.polytools.groebner`
+      or {meth}`~sympy.polys.polytools.GroebnerBasis.fglm`
```

Comment:
These methods are primarily used to solve *systems* of polynomial equations. Groebner bases are also a rather advanced thing to just drop someone into without more information, although unfortunately there aren't any guides about it in the docs that I know of. Also solve() calls groebner automatically, so you'd only really need to do this if you are an advanced user and know what you are doing.